src/services/bug/controller.ts

```diff
@@ -0,0 +1,35 @@
+import { StatusCodes } from 'http-status-codes';
+
+import { FastifyPluginAsync } from 'fastify';
+
+import { BUGS_RECIVER_EMAIL } from '../../utils/config';
+import { bugSchema } from './fluent-schema';
+import { generateEmail } from './utils';
+
+const plugin: FastifyPluginAsync = async (fastify) => {
+  fastify.post<{ Body: { name: string; email: string; details: string } }>(
+    '/',
+    {
+      schema: bugSchema,
+      preHandler: fastify.verifyAuthentication,
+    },
+    async (request, reply) => {
+      const { member, body: data } = request;
+      const { log, mailer } = fastify;
+
```

Comment:
I think you should check for a captcha here, to avoid spamming. You can find example in `auth/magicLink`.

Reply:
@LinaYahya this will require a bit more changes (in the frontend as well). Please create a new issue to handle this.